added resume qmd and job search blog post

joshyam-k opened the pull request.

Commit:
added resume qmd and job search blog post

simonpcouch commented on blog-posts/blog4.qmd:
```diff
@@ -4,3 +4,18 @@ description: ""
 editor: visual
 ---
+
+My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of that experience grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special lov ...
```
```suggestion
My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of those experiences grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special love for solving very technical problems, I care just as much about how those problems get solved.
```

simonpcouch replied:
I think that you ought to forefront that you're looking for a job within the first sentence or two. I think it might also be nice to see a glimpse of your professional identity independent of your degree. Another way this could flow:

"My name is Josh Yamamoto. I'm a statistical software developer with an appreciation for interpersonal connection and principled data science workflows. After graduation from Reed College this spring with a degree in Mathematics & Statistics, I'll be on the job market for roles in statistical programming and data science."

simonpcouch commented on blog-posts/blog4.qmd:
```diff
@@ -4,3 +4,18 @@ description: ""
+
+My name is Josh Yamamoto and I'm finishing up my final year as a Mathematics & Statistics major at Reed College. I have been lucky enough to interact with data science in a wide variety of settings and flavors. Out of that experience grew my primary interest: a deep passion for propagating thoughtfulness and intentionality in all of the ways that data science gets done. While I have a special lov ...
+
+I have a lot of research experience having done social justice computer science research at Reed College, biostatistics research at a joint program at Duke & NC State, and most recently forestry data science research at Harvard University. Each of these programs have been both deeply technical and deeply collaborative, giving me a strong appreciation for how important partnership is to solving di ...
```
```suggestion
My research experiences include social justice computer science research at Reed College, biostatistics research at a joint program at Duke & NC State, and most recently forestry data science research at Harvard University. Each of these programs have been both deeply technical and deeply collaborative, giving me a strong appreciation for how important partnership is to solving difficult problems. 
```

simonpcouch commented on blog-posts/blog4.qmd:
```diff
@@ -4,3 +4,18 @@ description: ""
+
+I have a lot of research experience having done social justice computer science research at Reed College, biostatistics research at a joint program at Duke & NC State, and most recently forestry data science research at Harvard University. Each of these programs have been both deeply technical and deeply collaborative, giving me a strong appreciation for how important partnership is to solving di ...
+
+I also have done work as a data science consultant at my college for the last three years, advising and implementing programs in R and other data technologies to catalyze research efforts by faculty and student researchers. I have also served as a course assistant and tutor for machine learning, and introductory statistics courses. While I always enjoy solving technical problems, I get just as mu ...
```
```suggestion
I also have done work as a data science consultant at my college for the last three years, advising and implementing programs in R and other data technologies to catalyze research efforts by faculty and student researchers. I have also served as a course assistant and tutor for machine learning and introductory statistics courses. While I always enjoy solving technical problems, I get just as much joy out of equipping other people to learn and solve problems themselves.
```

simonpcouch commented on blog-posts/blog4.qmd:
```diff
@@ -4,3 +4,18 @@ description: ""
+
+You can check out my complete resume [here](resume.qmd)!
+
+I'm looking for a job at a place where I can be a part of building tools for data scientists and statisticians. 
```
```suggestion
I'm looking for a role where I can be a part of building tools for data scientists and statisticians. 
```

simonpcouch replied:
This ought to come before the resume, and be a bit longer. I'm guessing you'd like to work remotely or in the Portland, OR area? How can folks get in touch with you? Are you open to chatting with folks starting now, and when is your desired start date (e.g. "June 2023")?

Also, "for" or "with" data scientists and statisticians?

simonpcouch commented on blog-posts/blog4.qmd:
```diff
@@ -4,3 +4,18 @@ description: ""
 date: "11/14/22"
```
I can't comment directly on the line, but I'd a bit more to the title and description (if it will show up on socials) here.

**I'm On The Job Market** - *Looking for roles in statistical software.*

simonpcouch commented on resume.qmd:
```diff
@@ -0,0 +1,67 @@
+
+**BA Mathematics with a Concentration in Statistics** \| Reed College, Portland, OR \| May 2023
+
+Coursework in Statistics, Computer Science, Mathematics, Physics, and Philosophy Overall GPA: 3.95; Concentration GPA: 4.00
```
Missing a `|` before "Overall"?

simonpcouch commented on resume.qmd:
```diff
@@ -0,0 +1,67 @@
+
+## Joshua Yamamoto
+
+**Portland, OR • github: joshyam-k • yamamojo\@reed.edu**
```
Since this is a `.qmd`, could link your GitHub explicitly. Also, maybe include an email that will persist after you graduate?

simonpcouch commented on resume.qmd:
```diff
@@ -0,0 +1,67 @@
+**Biostatistics Researcher** \| Duke University /North Carolina State University SIBS \| Virtual \| May--July 2021
+
+-   Collaborated with other researchers on novel and unique biostatistics research projects. These primarily involved different analysis and modeling methods within the frame of biostatistics
+-   Attended learning sessions with high level biostatistics researchers and professors
```
```suggestion
-   Attended learning sessions with renowned biostatistics researchers and professors
```

simonpcouch replied:
Not a big fan of my own suggestion, but "high level" feels a bit off here.

simonpcouch reviewed (commented):
This looks really great, Josh. :) I just made suggestions with small copy edits, and then added bigger-picture thoughts in comments.

joshyam-k committed:
Update resume.qmd

Co-authored-by: Simon P. Couch <simonpatrickcouch@gmail.com>

joshyam-k committed:
Update blog-posts/blog4.qmd

Co-authored-by: Simon P. Couch <simonpatrickcouch@gmail.com>

joshyam-k committed:
Update blog-posts/blog4.qmd

Co-authored-by: Simon P. Couch <simonpatrickcouch@gmail.com>

joshyam-k committed:
Update blog-posts/blog4.qmd

Co-authored-by: Simon P. Couch <simonpatrickcouch@gmail.com>

joshyam-k committed:
Update blog-posts/blog4.qmd

Co-authored-by: Simon P. Couch <simonpatrickcouch@gmail.com>

Commit:
added more changes!

Commit:
move

Merge branch 'job-search-blog' of https://github.com/joshyam-k/personal-website into job-search-blog

# Conflicts:
#	blog-posts/blog4.qmd
